spec/models/user_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.create(name: 'Gabriela', posts_counter: 1) }
+
+  before { subject.save }
+
+  it 'name must not be blank' do
+    subject.name = ''
+    expect(subject).to_not be_valid
+  end
+
+  it 'posts_counter must be an integer greater than or equal to zero.' do
+    subject.posts_counter = -1
+    expect(subject).to_not be_valid
+  end
+
+  it 'subject should be an instance of the class User' do
+    expect(subject).to be_instance_of User
+  end
+
+  it 'The method recent_posts should be available for the instance subject' do
+    expect(subject).to respond_to(:recent_posts)
+  end
+end
```

Comment:
- [ ] Good job with test coverage, but since we are aiming for a 100% test coverage and also it is a [requirement ](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md)for this project, " `Add unit specs for all of your models' methods and validations.` ", please kindly cover all the methods in your  `Comment and Like model` with test. You omitted those two models, please kindly cover them with tests.

Reply:
Agree, I will add the unit specs.